lesson_12/contact_us.html

```diff
@@ -47,7 +47,7 @@
     <!-- Intro -->
     <div class="intro">
         <h1 class="intro__heading">Your favorite language school</h1>
-        <button class="btn btn-intro">Order lesson</button>
+        <button type="submit" onclick="location.href='contact_us.html'" class="btn btn-intro">Order lesson</button>
```

Comment:
why so complicated?
you can use just `<a>`

Reply:
I've read about the difference between <button> and <a>. And then, I've chosen <button>. Is this wrong?